### ISSUE TITLE
Documentation for RealTimeAPI

### ISSUE THREAD

**Issue by inf3cti0n95** [inf3cti0n95/Rocket.Chat.RealTime.API.RxJS#53]
As there is a general lack of documentation for RealtimeAPI in  Rocket.Chat's official docs and this package also needs a better example based docs, a New, Better Documentation

**Comment by Boby** [comment]
yes, documentations on how to use? as I find it only connecting via rx and is it compatible with ionic 4?
